lms/envs/devstack.py

```diff
@@ -34,6 +34,24 @@
 for log_name, log_level in LOG_OVERRIDES:
     logging.getLogger(log_name).setLevel(log_level)
 
+#efischer local SQL inspection, don't merge me please!
+#reference: http://www.dabapps.com/blog/logging-sql-queries-django-13/
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': True,
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler'
+        }
+    },
+    'loggers': {
+        'django.db.backends': {
+            'level': 'DEBUG',
+            'handers': ['console']
+        }
+    }
+}
```

Comment:
In case anyone's curious, this will log your SQL calls to console as they happen. Pairing that with interactive debugging made this investigation very quick.